src/custom-element.js

```diff
@@ -119,6 +119,8 @@ export function upgradeOrRegisterElement(win, name, toClass) {
     const element = stub.element;
     if (element.tagName.toLowerCase() == name) {
       tryUpgradeElementNoInline(element, toClass);
+      // Remove element from array.
+      stubbedElements.splice(i--, 1);
```

Comment:
Yeah, I actually think this is more correct. `upgrade` isn't idempotent, so calling it again doesn't make any sense. We don't really expect errors, so it should be fine either way.